faucet/tasks.py

```diff
@@ -161,3 +161,28 @@ def update_donation_receipt_pending_status():
     )
     for pending_donation_receipt in pending_donation_receipts:
         process_donation_receipt.delay(pending_donation_receipt.pk)
+
+@shared_task
+def update_claims_count():
+    active_faucets = Faucet.objects.filter(is_active=True)
+    for faucet in active_faucets:
+        total_claims_since_last_round = ClaimReceipt.objects.filter(
+            faucet=faucet,
+            datetime__gte=RoundCreditStrategy.get_start_of_previous_round(),
+            _status__in=[ClaimReceipt.VERIFIED]
+        ).count()
+        faucet.total_claims_since_last_round = total_claims_since_last_round
+        faucet.save()
```

Comment:
you must separate  function body and scheduling logic look at the celery_tasks.py.

also you could make another function that get faucet_id and calculate total claim per faucet to add some concurrency.